docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -292,31 +299,70 @@ Next copy the public URL that ``ngrok`` is using, i.e. if the following is the o
+     Main page of the `grid` view.
 
-.. some general comment on importance of the graph?
-.. a sentence on how to continue from here
+Now we are going to look at the available band structure nodes, for which we will need to expand the `Array` lateral section and click on the `BandsData` subsection:
 
-Browse your AiiDA database:
+   .. figure:: include/screenshots/explore_01.png
+     :width: 100%
 
-    * Start by finding your `Quantum ESPRESSO`_ calculation (the type of node is called a ``CalcJobNode`` in AiiDA, since it is run as a job on a scheduler).
-    * Select ``Calculations`` in the left menu to filter for calculations only.
-    * Inspect the raw inputs and outputs of the calculation, and use the provenance browser to explore the input and output nodes of the calculation and the whole provenance of your simulations.
+     All nodes of type ``BandsData``, listed in the `grid` view.
+
+Here we can just select one of the available nodes and click on `details` on the right.
+This will take us to the `details` view of that particular node, which will have useful information on it.
+
+   .. figure:: include/screenshots/explore_02.png
+     :width: 100%
+
+     The `details` view of a specific node of type ``BandsData``.
+
+
+In this case we can see that the Explore Section has available a useful bands visualizer for nodes of type ``BandsData``.
+It also contains (as it does for all type of nodes) the `AiiDA Provenance Browser` on its right.
+This tool allows us to easily explore the connections between nodes more information about, for example, how these results were obtained.
+In effect, we can identify the ``CalcJob`` node that produced this result (red square with the link labeled ``output_band``) and click on it.
+This will immediately redirect us to the `details` page for that ``CalcJob`` node:
+
+   .. figure:: include/screenshots/explore_03.png
+     :width: 100%
+
+     The `details` view of the ``CalcJob`` node that created the original ``BandsData`` node.
+
+You can check out here the details of the calculation, the actual input and output files, the `Node metadata` and `Job information` dropdown menus, etc.
+You may also want to know to which crystal structure does its generated band structure corresponds to.
+Although this information can also be found inside the input files, we will look for it directly in the input nodes, again by using the `AiiDA Provenance Browser`.
+This time we will look for the ``StructureData`` node (green circle) that has an outgoing link (so, the arrow points from the ``data`` node to the central current ``process`` node) with the label `structure`.
+We will finally click in that node.
+
+   .. figure:: include/screenshots/explore_04.png
+     :width: 100%
+
+     The `details` view of the ``StructureData`` node that corresponds to the original ``BandsData`` node.
+
+We can see in this case that the original ``BandsData`` corresponds to a Silica structure.
```

Comment:
```suggestion
We can see that - as expected - the original ``BandsData`` was calculated to for a silicon structure.
```

Reply:
"As expected"? Why would they expect this was a Silica structure? Also, I'm using Silica specifically because that's the common name for [Silicon Dioxide](https://en.wikipedia.org/wiki/Silicon_dioxide), I didn't just mean a generic silicon structure. Why did you wanted to change that? Maybe it would be clearer if I just said "for Silica" instead of "for a Silica structure"? (although I'm not sure if it can have different phases...)

Reply:
I thought we were running silicon in the AiiDAlab demo (hence it was to be expected, since we're _only_ running silicon in this case? Maybe I misunderstood. @csadorf what is the plan for the demo?

Reply:
I plannes to run Si, but I can do SiO2 if that's less confusing.

Reply:
Perhaps it would be better, actually, since then they aren't rerunning the same structure in the QE part? (see point 3 in #286)

Reply:
I didn't know which one was being used so I did both just to have it populated here. However my idea was not that they did the exact same thing, but just to give some pointers/indications on how to browse the database. I tried to be explicit in the text (that's what you see a lot of "in this case", "for this whatever", "this is just an example") but I can try to go even a bit further and literally mention that they will not necessarily have this structure.

I don't think its critical to do this for the silicon since the explore interface really doesn't allow us to guarantee they will follow the same path, so in any case they might end up in a different starting structure (the only info in the table that can distinguish the different band structures is UUID and creation time: the first will be different, so does it make sense to rely on the second to indicate them which band structure will lead them to the same origin structure as what I show them??). I don't know, if you disagree I can try to get different screenshots for silicon.

BTW: I wouldn't change the silicon as the base one for the aiidalab demo, if the rest of the tutorial was done with that in mind, it might change the numbering on the pks (which is not critical since we don't rely on that, but its anyway a bit annoying).

Reply:
@mbercx @ramirezfranciscof Can you give me clear guidance on which structure I should use?

Reply:
@csadorf I'm running the SiO2 structure with the QE app now, and it seems to take quite some time on the default resources (I'm already running 7 minutes). Maybe best to stick with Si to keep the AiiDAlab demo short?

Reply:
> I didn't know which one was being used so I did both just to have it populated here. However my idea was not that they did the exact same thing, but just to give some pointers/indications on how to browse the database. 

Right, sorry, that was my assumption (also in other changes I suggested). I think it's fine to keep it like this, let's go ahead and merge. I'll go through it again once the sections have been split properly and maybe still make some small changes. 👍

Reply:
@mbercx Good point, SiO2 takes longer, so I'll stick to Si for the demo.